File: Week 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_pyramid(int height);

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    print_pyramid(height);

    return 0;
}

void print_pyramid(int height)
{
    int i, j, spaces;

    for (i = 1; i <= height; i++)
    {
        spaces = height - i;
        for (j = 1; j <= spaces; j++)
        {
            printf(" ");
        }
        for (j = 1; j <= height - spaces; j++)
        {
            printf("#");
        }
        printf("  ");
        for (j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 6/sentimental-mario-more/mario.py
def main():
    height = 0
    while height < 1 or height > 8:
        try:
            height = int(input("Height: ").strip())
        except (TypeError, ValueError):
            continue
    print_pyramid(height)


def print_pyramid(height):
    for i in range(height):
        spaces = " " * (height - i - 1)
        hashes = "#" * (i + 1)
        spaces_2 = "  "
        hashes_2 = "#" * (i + 1)
        print(spaces + hashes + spaces_2 + hashes_2)


main()
